src/executor/query.rs

```diff
@@ -140,6 +140,7 @@ impl<T: TryGetable> TryGetable for Option<T> {
         match T::try_get_by(res, index) {
             Ok(v) => Ok(Some(v)),
             Err(TryGetError::Null(_)) => Ok(None),
+            Err(TryGetError::DbErr(DbErr::Query(RuntimeErr::SqlxError(sqlx::Error::ColumnNotFound(_))))) => Ok(None),
```

Comment:
Hey @Reverier-Xu, thanks for contributing!! This is interesting!!

I think this will work if it's feature guarded as below.

```suggestion
#[cfg(feature = "sqlx-dep")]
Err(TryGetError::DbErr(DbErr::Query(RuntimeErr::SqlxError(sqlx::Error::ColumnNotFound(_))))) => Ok(None),
```

Also, can we add a few test cases to proof that it's actually working as expected? We might need to make some similar changes to the mock interface as well.